lib/server.js

```diff
@@ -1,18 +1,19 @@
 ReactiveTable = {};
 
 ReactiveTable.publish = function (name, collectionOrFunction, selectorOrFunction, settings) {
-    Meteor.publish("reactive-table-" + name, function (publicationId, filters, fields, options, rowsPerPage) {
+    Meteor.publish("reactive-table-" + name, function (publicationId, filters, fields, options, rowsPerPage, serverArgs) {
       check(publicationId, String);
       check(filters, [Match.OneOf(String, Object)]);
       check(fields, [[String]]);
       check(options, {skip: Match.Integer, limit: Match.Integer, sort: Object});
       check(rowsPerPage, Match.Integer);
+      check(serverArgs, Match.Optional(Match.Any));
 
       var collection;
       var selector;
-      
+
       if (_.isFunction(collectionOrFunction)) {
-        collection = collectionOrFunction.call(this);
+        collection = collectionOrFunction.call(this, serverArgs);
```

Comment:
If the name is going to be `serverArgs`, I think they should be provided as an array and the code should call `collectionOrFunction.apply`. Overall I think that this is a better solution than naming the attribute `serverArg` and having a restriction of one argument.